src/components/offer-card/OfferCard.jsx

```diff
@@ -0,0 +1,95 @@
+import { t } from 'i18next'
+
+import Rating from '@mui/material/Rating'
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import LanguageIcon from '@mui/icons-material/Language'
+import IconButton from '@mui/material/IconButton'
+import TurnedIn from '@mui/icons-material/TurnedIn'
+import TurnedInNot from '@mui/icons-material/TurnedInNot'
+
+import AppButton from '~/components/app-button/AppButton'
+import AppChip from '~/components/app-chip/AppChip'
+
+import { styles } from './OfferCard.styles'
+
+const OfferCard = ({
+  imgSrc,
+  rating,
+  name,
+  bio,
+  description,
+  languages,
+  price,
+  isBookmarked,
+  subject,
+  level
+}) => {
+  return (
+    <Box sx={ styles.container }>
+      <Box sx={ styles.leftContainer }>
+        <Avatar src={ imgSrc } sx={ styles.avatar } />
+        <Box sx={ styles.ratingContainer }>
+          <Rating defaultValue={ Math.round(rating) } readOnly sx={ styles.rating } />
+          <Typography>{ rating }</Typography>
+        </Box>
+      </Box>
+
+      <Box sx={ styles.middleContainer }>
+        <Typography variant='h6'>{ name }</Typography>
+        <Typography sx={ styles.bio }>{ bio }</Typography>
```

Comment:
better use titleWithDescription component for name and bio

Reply:
I don't think so.